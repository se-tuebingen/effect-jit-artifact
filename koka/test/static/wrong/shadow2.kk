// shadow parameter
val id : forall<a,a> (a) -> a = (fn(x) {
  return x
})
