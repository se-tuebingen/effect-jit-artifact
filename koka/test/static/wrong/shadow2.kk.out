test/static/wrong/shadow2@kk(2,19): kind error: Type a is already defined